src/main/java/io/snyk/jenkins/tools/SnykInstaller.java

```diff
@@ -135,19 +135,9 @@ public boolean isApplicable(Class<? extends ToolInstallation> toolType) {
   private static class GetPlatform extends MasterToSlaveCallable<Platform, IOException> {
     private static final long serialVersionUID = 1L;
 
-    private final String nodeDisplayName;
-
-    GetPlatform(String nodeDisplayName) {
-      this.nodeDisplayName = nodeDisplayName;
-    }
-
     @Override
-    public Platform call() throws IOException {
-      try {
-        return Platform.current();
-      } catch (ToolDetectionException ex) {
-        throw new IOException(format("Could not determine platform on node %s", nodeDisplayName));
-      }
+    public Platform call() {
+      return Platform.current();
```

Comment:
We can let the exception bubble up and see the stacktrace if needed. No need to catch and rethrow. Less code to maintain.